aries-site/src/examples/components/card/NavigationalAnatomy.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  Diagram,
+  Grid,
+  Heading,
+  Paragraph,
+  Stack,
+} from 'grommet';
+import { Notes } from 'grommet-icons';
+import { Annotation } from '../../../layouts';
+
+const DashedBox = ({ ...rest }) => (
+  <Box
+    alignSelf="start"
+    border={{ style: 'dashed' }}
+    round="xxsmall"
+    {...rest}
+  />
+);
+
+const CardExample = ({ ...rest }) => (
```

Comment:
Are we not providing the flexibility to have a subtitle for navigational cards?

Reply:
Good call -- will add the subtitle into this.